src/edit/getEXIFdata.js

```diff
@@ -0,0 +1,85 @@
+L.EXIF = function getEXIFdata(img) {
```

Comment:
This is super. Shall we move it into a subfolder like `/src/edit/tools/` as the first step towards https://github.com/publiclab/Leaflet.DistortableImage/issues/161? But we can do that in follow-up.